storage/StorageManager.cpp

```diff
@@ -687,8 +687,13 @@ StorageManager::BlockHandle StorageManager::loadBlockOrBlob(
 #ifdef QUICKSTEP_DISTRIBUTED
     const string domain_network_address = getPeerDomainNetworkAddress(BlockIdUtil::Domain(block));
     DLOG(INFO) << "Pulling Block " << BlockIdUtil::ToString(block) << " from " << domain_network_address;
+
+    // Customize the grpc channel
+    grpc::ChannelArguments channelArgs;
+    channelArgs.SetMaxReceiveMessageSize(kGrpcChanelSize);
```

Comment:
Please add a condition wrapper based on the `GRPC` version number for this API call, since not every version has this API.